seq2science/workflows/rna_seq/Snakefile

```diff
@@ -7,13 +7,13 @@ include: "../../rules/configuration.smk"
 # load the remaining relevant rules
 include: f"{config['rule_dir']}/alignment.smk"
 include: f"{config['rule_dir']}/bam_cleaning.smk"
-include: f"{config['rule_dir']}/bigfiles.smk"
 include: f"{config['rule_dir']}/deseq2.smk"
 include: f"{config['rule_dir']}/gene_counts.smk"
 include: f"{config['rule_dir']}/get_genome.smk"
 include: f"{config['rule_dir']}/get_fastq.smk"
 include: f"{config['rule_dir']}/merge_replicates.smk"
 include: f"{config['rule_dir']}/strandedness.smk"
+include: f"{config['rule_dir']}/bigfiles.smk"
```

Comment:
This isn't necessary right? At least I don't see why it would be. It's extremely minor, but nice to keep them alphabetical when possible

Reply:
:zipper_mouth_face: